guides/common/modules/proc_verifying-dns-resolution.adoc

```diff
@@ -9,33 +9,30 @@ Verify the full forward and reverse DNS resolution using a fully-qualified domai
 +
 [options="nowrap"]
 ----
-# ping -c1 localhost
-# ping -c1 `hostname -f` # my_system.domain.com
+# dig localhost
+# dig `hostname -f` # my_system.domain.com
 ----
 +
-Successful name resolution results in output similar to the following:
+When name resolution succeeds, the output resembles the following:
 +
 [options="nowrap"]
 ----
-# ping -c1 localhost
-PING localhost (127.0.0.1) 56(84) bytes of data.
-64 bytes from localhost (127.0.0.1): icmp_seq=1 ttl=64 time=0.043 ms
-
---- localhost ping statistics ---
-1 packets transmitted, 1 received, 0% packet loss, time 0ms
-rtt min/avg/max/mdev = 0.043/0.043/0.043/0.000 ms
-
-# ping -c1 `hostname -f`
-PING hostname.gateway (XX.XX.XX.XX) 56(84) bytes of data.
-64 bytes from hostname.gateway (XX.XX.XX.XX): icmp_seq=1 ttl=64 time=0.019 ms
-
---- localhost.gateway ping statistics ---
-1 packets transmitted, 1 received, 0% packet loss, time 0ms
-rtt min/avg/max/mdev = 0.019/0.019/0.019/0.000 ms
-
+# dig localhost
+; <<>> DiG 9.18.30 <<>> localhost
+...
+;; ANSWER SECTION:
+localhost.		0	IN	A	127.0.0.1
+...
+
+# dig  `hostname -f`
+; <<>> DiG 9.18.30 <<>> my_system.domain.com
+...
+;; ANSWER SECTION:
+my_system.domain.com.			0	IN	A	XXX.XXX.X.XX
+my_system.domain.com.			0	IN	A	XX.XX.XXX.X
 ----
 
-. To avoid discrepancies with static and transient host names, set all the host names on the system by entering the following command:
+. To avoid discrepancies with static and transient host names, set all the host names on the system with the following command:
```

Comment:
This might work even better:
```suggestion
. To avoid discrepancies with static and transient host names, set all the host names on the system:
```